### ISSUE TITLE
Strange phenomenon during my experiment

### ISSUE THREAD

**Issue by feixuetuba** [hizhangp/yolo_tensorflow#34]
@hizhangp I have use the pascal_voc2007 dataset to train the network, and try to use the trained network to do some prediction. I found that whatever image feed to the network, It always generate the same output(both bonding-box and confidence in different image are the same), through the total_loss is quite low.
     So I review the code, and try to understand what have happened during the  training progress, and There are two place is really hard for me to get it through, they both in "loss_layer"
      First of all, “boxes = tf.tile(boxes, [1, 1, 1, self.boxes_per_cell, 1]) / self.image_size”， why the "boxes" needed to be divided by "image_size" , when the predict_boxes_train only divided by cell_size
      Secondly, why the predict_box should be plused with "offset" when we calculate the predict_boxes_train,?can we remove it?

**Comment by ddsrt-sean** [comment]
Hello， Sir
I used pascal_voc2007 dataset to train the network with default config. I tried it many times  but the Loss always is keeping around 11。the default ckpt(YOLO_small.ckpt) train the network, the loss is 4。
Can you tell me the loss you got with default config, and you can help me to reduce loss ？
Thanks You!

**Comment by BlankAix** [comment]
@feixuetuba  Have you figured out the problem? 
I hope you can give me some advice about this.

**Comment by lei9wang** [comment]
@ddsrt-sean , how about your test result with your trained weights?
I also trained the network with default config, the loss is about 9. but there is not effect when I test it with my trained weights.
I don't know why, is it loss too big?

**Comment by guker** [comment]
> Hello， Sir
> I used pascal_voc2007 dataset to train the network with default config. I tried it many times but the Loss always is keeping around 11。the default ckpt(YOLO_small.ckpt) train the network, the loss is 4。
> Can you tell me the loss you got with default config, and you can help me to reduce loss ？
> Thanks You!

I have same problem，and  you must pretrain，not train from scrath，otherwise，loss is about 10.0 and does not decrease

**Comment by guker** [comment]
> @ddsrt-sean , how about your test result with your trained weights?
> I also trained the network with default config, the loss is about 9. but there is not effect when I test it with my trained weights.
> I don't know why, is it loss too big?

check https://github.com/hizhangp/yolo_tensorflow/issues/18#issue-229843655
